scripts/bungie.coffee

```diff
@@ -239,7 +240,7 @@ makeRequest = (bot, endpoint, callback, params) ->
   queryParams = if params then '?'+params else ''
   url = baseUrl+endpoint+trailing+queryParams
 
-  console.log("making request: #{url}")
+  # console.log("making request: #{url}")
```

Comment:
I like having this for debugging purposes. but I can uncomment it after merging if need be